app/services/geoblacklight_event_processor/processor.rb

```diff
@@ -19,6 +19,10 @@ def doc
       event['doc']
     end
 
+    def bulk?
```

Comment:
For the purposes of my own understanding, this depends upon a Figgy server environment var. specifying that only bulk updates are used, correct?

Please see https://github.com/pulibrary/figgy/blob/master/app/services/event_generator/geoblacklight_event_generator.rb#L65 for reference.

Reply:
Yes. To reindex Pulmap, you would start a console session with `BULK=true` and run the the reindexer service.

https://github.com/pulibrary/figgy/blob/master/app/services/geo_resource_reindexer.rb

There's not a Pulmap reindex job right now, but there probably should be.

Reply:
If this is indeed the case, then I can create an issue in https://github.com/pulibrary/princeton_ansible before merging this (unless this should be controlled in the Capistrano config. for Figgy).

Reply:
Oh, sorry I misread your comment. You only set the BULK var when doing a full reindex.

Reply:
Normal atomic updates and deletes are unchanged.